test/useGet.test.ts

```diff
@@ -519,7 +519,8 @@ describe('Get composition', () => {
       expect(serviceOff).toHaveBeenCalledWith('patched', expect.anything());
       expect(serviceOff).toHaveBeenCalledWith('removed', expect.anything());
     });
-    it('should not unmount the event handlers when desired', () => {
+
+    it('should unmount the event handlers when desired', () => {
```

Comment:
I think it might be best to keep a test for `should not unmount the event handlers when ...`. This will also clarify how to use the composition when you want to NOT unmount the event handlers (I'm a bit confused about how to do that explicitly right now without the `disableUnloadingEventHandlers` param)

This test for the `unload` function is important too though.

Reply:
Thought about such a test as well, but how to test that a listener does not get released while nothing happens. Just a timeout? The `getCurrentInstance` check will return `null` in case you called the composition from outside of a components `setup` function, so there should be no need to disable it manually anymore.

Reply:
```ts
// composition
export const useEntity = () => {
  const findRes = useFind();
  return { findRes };
} 
```

```ts
// component
setup() {
  useEntity();
}
```

This is how we are using `useFind` a lot. This call is inside the setup function right?

So to adapt to the new version I would have to change the composition to the following?

```ts
const findRes = useFind();
// composition
export const useEntity = () => {
  return { findRes };
} 
```

Reply:
> This is how we are using useFind a lot. This call is inside the setup function right?

Right. All functions directly called by `setup` should be fine.

---

```ts
// composition
const findRes = useFind();
export const useEntity = () => {
  return { findRes };
} 
```

This should be the proper way of doing it in general, but it should be independent from the change.

Reply:
I find this very implicit. Maybe we can avoid the breaking change by keeping `disableUnloadingEventHandlers`